src/room/InCallView.tsx

```diff
@@ -99,17 +98,19 @@ export function ActiveCall(props: ActiveCallProps) {
   const livekitRoom = useLiveKit(props.userChoices, {
     sfuUrl: Config.get().livekit!.server_url,
     jwtUrl: `${Config.get().livekit!.jwt_service_url}/token`,
-    roomName: props.matrixInfo.roomName,
+    roomName: `${props.matrixInfo.roomName}(${props.groupCall.groupCallId})`,
```

Comment:
That's still the room name, rather than the room ID :D

Reply:
Sorry, I messed the things up. I've just rebased the PR against `lint-fixes` to contain the newest changes that were rebased against `livekit`. Now it should finally be fine 🙂